docs/cookbook/getFirstTuesdayOfMonth.mjs

```diff
@@ -0,0 +1,26 @@
+import assert from "assert";
+
+/**
+ * Gets the first tuesday of the month and returns its date
+ *
+ * @param {Temporal.YearMonth} queriedMonth YearMonth instance to query
+ * @returns {Temporal.DateTime} DateTime Instance which gives first tuesday
+ */
+function getFirstTuesday(queriedMonth) {
+    // We first need to convert to a date
+    let date = queriedMonth.withDay(1);
+
+    // Start at the beginning of the month then stop at the first tuesday
+    while (date.dayOfWeek !== 2) {
+        date = date.plus({ days: 1 });
+    }
+
+    return date;
+}
+
+const myMonth = Temporal.YearMonth.from("2020-02");
+const firstTuesdayOfMonth = getFirstTuesday(myMonth);
+assert.equal(
+    firstTuesdayOfMonth.toLocaleString("en-UK", { weekday: "long" }),
```

Comment:
I'd assert an ISO string but maybe `console.log` the nicely formatted string, or something like that. (This assertion fails for me anyway, I get `'Tuesday, 2/4/2020'`.)

Consider also asserting that `firstTuesdayOfMonth.dayOfWeek` equals 2.